change/@fluentui-web-components-b0669382-c509-4801-bd44-9849e8570a92.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30090&commitId=9535c4ad17188efbcb3e81a7b9aff6b113184840&env=prod&clientType=FLUENTUI)


  <details><summary><b>Tabs</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tabs.Size Medium Vertical.normal.chromium.png|30953| Changed|
|Tabs.Size Small Vertical - Dark Mode.normal.chromium.png|3377| Changed|
|Tabs.Size Large Vertical - Dark Mode.normal.chromium.png|6551| Changed|
|Tabs.Size Medium Vertical - Dark Mode.normal.chromium.png|5088| Changed|
|Tabs.Vertical.normal.chromium.png|30953| Changed|
|Tabs.Size Large Vertical.normal.chromium.png|39762| Changed|
|Tabs.Size Small Vertical.normal.chromium.png|19375| Changed|
|Tabs.Navigation Vertical.chromium.png|641| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>